server/routes/request-schema.ts

```diff
@@ -0,0 +1,14 @@
+import { check } from 'express-validator'
+
+const regExp = /^[A-Za-z0-9]*$/
+const schema = [
+  check('prisonerNumber').not().isEmpty({ ignore_whitespace: true }),
+  check('prisonerNumber').isLength({ min: 1 }),
+  check('prisonerNumber').not().isNumeric(),
```

Comment:
I think we should replace 
```
  check('prisonerNumber').not().isNumeric(),
  check('prisonerNumber').not().isAlpha(),
  check('prisonerNumber').custom(value => {
    return regExp.test(value.trim())
  }),
  ```
  With just `check('prisonerNumber').isAlphanumeric(),`
  
  we're just trying to check that no one is inserting any paths like `123/documents` or something, we don't really need to check that it looks like a real nomis ID (and prisoner number formats could change in the future)

Reply:
I tried this earlier, when we use isAlphanumeric, it still hit the API when only characters or only number given. so  i go for separate function